doc/dev/developer_guide/running-tests-locally.rst

```diff
@@ -52,12 +52,34 @@ Running your first test
 The Python tests in Ceph repository can be executed on your local machine
 using `vstart_runner.py`_. To do that, you'd need `teuthology`_ installed::
 
+Or you can do::
```

Comment:
I think since line 53 has a :: maybe you need to remove one colon there?

Reply:
That's not a mistake, that's how RST works.

Reply:
No, the :: on line 53 signals that a literal block is expected.

Reply:
Yeah, you are right. There's 2 `::`, which is not necessary.  It was a good catch. :)

I've fixed this issue and also modified my previous comment.

Reply:
Marking this as resolved.